Blizzard.IO.RabbitMQ/Entities/Rpc/RpcConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Blizzard.IO.RabbitMQ.Entities.Rpc
+{
+    public class RpcConfiguration
+    {
+        // Im not sure if to geive the user a function or to give a static string to return
```

Comment:
I need you for an explanation on this one ;)

Reply:
In the way you implemented it the user is able to define a way to generate eachange & queue names by `Type`.
I think its more flexible

Reply:
🚀